app/controllers/circles_controller.rb

```diff
@@ -10,20 +10,45 @@ def show
 
   def new
     @circle = Circle.new
+    @circle.build_circle_time
+    @day_of_weeks = DayOfWeek.all
   end
 
   def edit
   end
 
   def create
+    @circle = Circle.new(circle_params)
+    @circle.build_circle_image.build_image.image.attach(params[:circle][:image])
```

Comment:
[Q]
こんな感じだとダメなんだっけ？
(あとちゃんとみたら、circle_imageにimageをattachする感じでいいなと思った)
```suggestion
    @circle.circle_image.image.image.attach(params[:circle][:image])
```

Reply:
もし、これしか動かないんだったらimageモデルかcircle_imageモデルにメソッド作ってもいいかも。(参照の値渡しなので問題なし)

Reply:
なるほど！